bot/controllers/channels.js

```diff
@@ -0,0 +1,111 @@
+module.exports = () => {
+  const util = require('apex-util');
+
+  const _run = (message) => {
+    const ctrls = [
+      {
+        cmd: '!channels',
+        example: '!channels',
+        title: 'List All Channels',
+        desc: 'List all Armada Channels',
+        showWithHelp: true,
+        posTargetUser: null,
+        posSecondaryCmd: null,
+        regexSplitCharacter: null,
+        allowInDM: false,
+        resType: 'dm',
+        action: (message) => {
+          const channels = [];
+          util.log('help!', message);
+          message.guild.channels.map(channel => channels.push(channel.name));
+          return 'List of all Armada Channels: \n\n' + channels.join('\n');
+        },
+      },
+      {
+        cmd: '!announce',
+        example: '!announce <channel_name>,<channel_name>,<channel_name> <message>',
+        title: 'Announce to Channels',
+        desc: 'Broadcast to multiple specific channels. Channels are case-sensitive.',
+        showWithHelp: true,
+        posTargetUser: null,
+        posSecondaryCmd: null,
+        regexSplitCharacter: null,
+        allowInDM: false,
+        resType: 'dm',
+        action: (message, ctrl, msg) => {
+          const channels = msg.parsed[1].split(',');
+          util.log('Multiple Channels Parsing', channels, 4);
+
+          channels.map((channel) => {
+            const targetChannel = message.guild.channels.find('name', channel);
+            util.log('Asking API for Channel', targetChannel, 4);
+
+            if (targetChannel === null) {
+              return '"' + channel + '" is not a known channel. Try `!channels` to get a list of all Channels (They are case-sensitive)';
+            }
+            let msgParsedIndex = 2;
+            let preparedMessage = '';
+            while (msgParsedIndex < msg.parsed.length) {
```

Comment:
Consider commenting this loop to add info on what is being done. Comments should make the source code skimmable for other devs.